tasks/install.yml

```diff
@@ -0,0 +1,53 @@
+---
+
+# Configure nodejs
```

Comment:
Peertube is not installed here yet, the file name is confusing. Rename the file to something like `install_dependencies.yml`